rq/cli/cli.py

```diff
@@ -227,6 +227,11 @@ def worker(
     sentry_debug = sentry_debug or settings.get('SENTRY_DEBUG')
     sentry_dsn = sentry_dsn or settings.get('SENTRY_DSN')
     name = name or settings.get('NAME')
+    dict_config = options.get("dict_config", settings.get('DICT_CONFIG'))
```

Comment:
`dict_config` is never passed through `options`, right?

Reply:
I was trying to be consistent with the lines above.

Reply:
But if `dict_config` is never passed through `options`, I don't think we need that. It's more important to be correct than to be consistent :)